tests/storage/test_extraction.py

```diff
@@ -40,18 +42,35 @@ class ErtConfigBuilder:
     def __init__(self):
         self.ensemble_size = 1
         self._priors = {}
+        self._obs = []
+
+    def add_general_observation(self, observation_name, response_name, data):
+        """Add GENERAL_OBSERVATION
+
+        The `data` parameter is a pandas DataFrame. This is to be a two-column
+        frame where the first column are the values and the second column are
+        the errors. The index-column of this frame are the observation's indices
+        which link the observations to the responses.
+
+        """
+        self._obs.append((observation_name, response_name, data.copy()))
 
     def add_prior(self, name, entry):
         assert name not in self._priors
         self._priors[name] = entry
         return self
 
-    def build(self, path=None):
+    def build(self, path=None, path_executable=None):
         if path is None:
             path = Path.cwd()
-        (path / "JOB").write_text("EXECUTABLE /usr/bin/true\n")
+
+        if path_executable is None:
```

Comment:
Job generating code can be moved to `_build_job`, which:
1. Creates a file called `JOB`, pointing to `/usr/bin/true` by default, or to the locally created executable if `self._job_script` is set. The `self._job_script` lets us specify the code for a job to be run.
2. Appends "INSTALL_JOB" and "SIMULATION_JOB" to the ERT config file.